foundation/foundation-upgrade/src/test/resources/specifications/v1_7_0/foundation-core-java-migration.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Foundation Core Java upgrade migration
+
+  Scenario: Upgrade Foundation Core Java upgrade migration
```

Comment:
A: Scenario titles all being the same is odd. Each scenario should be titled such that it clearly conveys what specific case is being covered by the test.

Reply:
Updated the titles in the new migration feature file.